src/backend/tty.rs

```diff
@@ -583,6 +626,41 @@ impl Tty {
         );
         debug!("connecting connector: {output_name}");
 
+        let device = self.devices.get_mut(&node).context("missing device")?;
```

Comment:
Are any changes needed to `on_output_config_changed()`?

Reply:
No, but I switched `non_desktop_connectors` to be a HashSet so that when the config is reloaded the same connector isn't added multiple times.